app/controllers/ATSPAYEController.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import play.api.mvc.Action
+import services.NpsService
+
+import uk.gov.hmrc.play.microservice.controller.BaseController
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+object ATSPAYEDataController extends ATSPAYEDataController {
+  override val npsService = NpsService
+}
+
+trait ATSPAYEDataController extends BaseController {
```

Comment:
Trait/Object name should correspondent with file name. Intellij should be hightlighting this as a warning. Please align the names.